### PR TITLE
[FR] People

### PR DESCRIPTION
**Pull request by GTva**
French translation of osu! wiki : People page.

### CODE REVIEW

File: wiki/People/fr.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# L'Équipe
+<!-- There is no mention of the Staff Log article here, despite this being an index article above it -->
+
+Les groupes d'utilisateurs suivants représentent les membres de la communauté qui aident à maintenir osu!. La plupart de ces groupes peuvent être reconnus grâce à la couleur de leur pseudonyme ou de leur texte dans le chat, leur photo de profil et/ou le(s) badge(s) présent(s) sur leur profil.
+
+- [Beatmap Nominators](/wiki/People/Beatmap_Nominators)
+- [Contributeurs](/wiki/People/Community_Contributors)
+- [Développeurs](https://osu.ppy.sh/groups/11) <!-- This should be its own page under People -->
+- [Équipe de de modération globale](/wiki/People/Global_Moderation_Team)
```
Comment (RedNesto) [inline wiki/People/fr.md]:
```suggestion
- [Équipe de modération globale](/wiki/People/Global_Moderation_Team)
```

File: wiki/People/fr.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+- [Modérateurs par langue](/wiki/People/Language_Moderators)
+- [Nomination Assessment Team](/wiki/People/Nomination_Assessment_Team)
+- [osu! Alumni](/wiki/People/osu!_Alumni)
+- [Équipe Project Loved](/wiki/People/Project_Loved_Team)
+- [Support technique](/wiki/People/Support_Team)
+- [L'administration](/wiki/People/The_Team)
```
Comment (RedNesto) [inline wiki/People/fr.md]:
```suggestion
- [Administration](/wiki/People/The_Team) 
```

Pour rester cohérent avec les autres éléments de la liste.

**Comment by Repflez** [comment]
Your PR has several commits with non-descriptive messages.

Commit messages are used to learn about changes to a specific file/article without having to open the commit directly on GitHub or other tools used by contributors.

Please look into [changing your message to be more descriptive](https://osu.ppy.sh/help/wiki/osu!_wiki_Contribution_Guide/Common_Issues#someone-told-me-to-use-meaningful-commit-messages!), you still have time to fix them.

File: wiki/People/fr.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# L'Équipe
+<!-- There is no mention of the Staff Log article here, despite this being an index article above it -->
```
Comment (cl8n) [inline wiki/People/fr.md]:
```suggestion
```
this comment has to be addressed in en.md, so it's fine to remove it from here

File: wiki/People/fr.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+Les groupes d'utilisateurs suivants représentent les membres de la communauté qui aident à maintenir osu!. La plupart de ces groupes peuvent être reconnus grâce à la couleur de leur pseudonyme ou de leur texte dans le chat, leur photo de profil et/ou le(s) badge(s) présent(s) sur leur profil.
+
+- [Beatmap Nominators](/wiki/People/Beatmap_Nominators)
```
Comment (cl8n) [inline wiki/People/fr.md]: can you alphabetize this list according to French rules?

**Comment by cl8n** [comment]
I updated this with new stuff from en.md, can you please translate line 3 @GTva 
can merge after that

File: wiki/People/fr.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
+# L'Équipe
+
+*For a list of promotions and retirements, see: [Staff Log](/wiki/People/Staff_Log)*
```
Comment (UltayJr) [inline wiki/People/fr.md]:
```suggestion
*Afin de trouver la liste des promotions et des retraites, veillez consulter : [Archives de l'équipe](/wiki/People/Staff_Log)*
```

Comment (UltayJr) [reply wiki/People/fr.md]: Just need to be accepted and that's merged !

Comment (cl8n) [reply wiki/People/fr.md]: thank you

**Pull request merged**